driver/pace/driver/tools.py

```diff
@@ -25,10 +31,14 @@ def command_line(action: str, sdfg_path: Optional[str], report_detail: Optional[
     """
     Run tooling.
     """
-    if action == ACTION_SDFG_MEMORY_COUNT:
+    if action == ACTION_SDFG_MEMORY_STATIC_ANALYSIS:
+        if sdfg_path is None or not os.path.exists(sdfg_path):
```

Comment:
we can just make it a required arg, right?

Reply:
Sure. I was keeping it open for other kinds of tools (non SDFG ones), but truly it's not needed to be so generic. Will change